packer/run_interfaces.go

```diff
@@ -27,9 +27,15 @@ type Evaluator interface {
 	EvaluateExpression(expr string) (output string, exit bool, diags hcl.Diagnostics)
 }
 
+type InitializeOptions struct {
+	// When set, GetBuilds will skip the execution of datasources and will use
+	// the placeholder provided by the datasource itself for validation only.
+	ValidationOnly bool
```

Comment:
Would it be a good idea to make this more explicit ?
```suggestion
	ExecuteDatasources bool
```

Reply:
I think the datasource execution should be the default here, then maybe `SkipDatasourceExecution` 🤔